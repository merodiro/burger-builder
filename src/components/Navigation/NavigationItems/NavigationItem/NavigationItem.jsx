import React from 'react'

import classes from './NavigationItem.module.css'

const NavigationItem = ({ active, link, children }) => (
  <li className={classes.NavigationItem}>
    <a href={link} className={active && classes.active}>
      {children}
    </a>
  </li>
)

export default NavigationItem
