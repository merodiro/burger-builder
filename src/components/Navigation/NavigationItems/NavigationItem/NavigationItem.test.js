import React from 'react'
import ReactDOM from 'react-dom'
import { shallow } from 'enzyme'

import NavigationItem from './NavigationItem'

describe('<NavigationItem>', () => {
  it('renders the same output', () => {
    const component = shallow(
      <NavigationItem link="/" active>
        A link
      </NavigationItem>
    )
    expect(component).toMatchSnapshot()
  })
})
