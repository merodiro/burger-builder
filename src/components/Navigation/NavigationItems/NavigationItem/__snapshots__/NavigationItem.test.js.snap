// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavigationItem> renders the same output 1`] = `
<li
  className="NavigationItem"
>
  <a
    className="active"
    href="/"
  >
    A link
  </a>
</li>
`;
